internal/core/src/index/IndexFactory.cpp

```
@@ -173,11 +173,9 @@ IndexFactory::VecIndexLoadResource(
                     index_type, index_version, config);
             break;
         default:
-            PanicInfo(
-                milvus::DataTypeInvalid,
-                fmt::format(
-                    "invalid data type to estimate index load resource: {}",
-                    field_type));
+            LOG_ERROR("invalid data type to estimate index load resource: {}",
```

Better to throw an exception instread of return success.